content/blog/2025/11/2025-11-07-twim.md

```diff
@@ -0,0 +1,265 @@
+> 
+> #### We're also on Matrix Live this week!
+> 
+> 🎤 📣 📺 We just had the opportunity to talk about NeoBoard on this week's Matrix Live, so if you're reading this on the matrix.org blog, check out the top of this TWIM article to watch the interview video!
+
+## Dept of SDKs and Frameworks 🧰
+
+### Polyjuice ([website](https://gitlab.com/polyjuice))
+
+Elixir libraries related to the Matrix communications protocol.
+
+[uhoreg](https://matrix.to/#/@hubert:uhoreg.ca) reports
+
+> After a too-long hiatus, I've started working on the [Polyjuice Project](https://gitlab.com/polyjuice) again, a set of libraries and tools for Matrix written in Elixir.
+>
+> I've updated the room version support in Polyjuice Utils (a library for functions common to clients and servers) to room version 11 with the help of Ben W.  Room version support in Polyjuice Utils includes identifier parsing, redaction rules, and authorization checks.  Room version 12 should be coming soon.
+> 
+> I've also started adding API definitions (such as endpoint paths and schema definitions) to Polyjuice Utils, and updating Polyjuice Client and Polyjuice Server to use those API definitions.  This should reduce code duplication between Polyjuice Client and Polyjuice Server, and make things more consistent.  The current goal is to implement API definitions for at least all the endpoints and types that Polyjuice Client and Polyjuice Server currently use.  This will probably take a while, and means that I won't be implementing much new stuff while I work on this, but I think it will be worth the effort.
+>
+> Aside from those changes, I will be dropping support for older versions of Elixir.  The goal is try to support the version of Elixir that is in Debian oldstable, and as such, Elixir versions older than 1.14 will no longer be supported.
+
+## Dept of Ops 🛠
+
+[spaetz](https://matrix.to/#/@spaetz:sspaeth.de) says
+
+> [testmatrix](https://codeberg.org/spaetz/testmatrix) (a synapse sanity checker) has a new release (v0.2). Minor improvements are:
+> 
+> * Test for MSC 4108 (QR code login)
+> * Check if public room directory is enabled
+> * Warn if client well-known baseurl does not start with https://
+> * a few other minor bugfixes and improvements
+
+## Dept of Events and Talks 🗣️
+
+### FOSDEM 2026 Decentralised Communications Devrooom
+
+[Matthew](https://matrix.to/#/@matthew:matrix.org) says
+
+> Excited to announce that we proposed a Decentralised Devroom for FOSDEM 2026 as a change from the usual Matrix devroom, to reflect FOSDEM's emphasis this year on cross-project collaboration rather than per-project devrooms.  The devroom has been accepted, and we've now issued a Call For Participation here: <https://matrix.org/blog/2025/11/fosdem-cfp/> - so you have **3 weeks to submit talks** on the latest and great projects you'd like to present.  So please get thinking and submitting (especially if you want Matrix to be represented in the devroom! :D)
```

Comment:
```suggestion
> Excited to announce that we proposed a Decentralised Devroom for FOSDEM 2026 as a change from the usual Matrix devroom, to reflect FOSDEM's emphasis this year on cross-project collaboration rather than per-project devrooms.  The devroom has been accepted, and we've now issued a Call For Participation here: [https://matrix.org/blog/2025/11/fosdem-cfp/](@/blog/2025/11/2025-11-04-fosdem-cfp.md) - so you have **3 weeks to submit talks** on the latest and great projects you'd like to present.  So please get thinking and submitting (especially if you want Matrix to be represented in the devroom! :D)
```